README.md

```diff
@@ -290,12 +322,12 @@ Be aware that by storing it in `~/.profile` or similar you will make it availabl
 
 ### Documentation
 
-You can use `aegir docs` to generate documentation. This uses [documentation.js](http://documentation.js.org/) with the theme [clean-documentation-theme](https://github.com/dignifiedquire/clean-documentation-theme).
+You can use `aegir docs` to generate documentation, this command uses `aegir ts --preset docs` internally.
 
 To publish the documentation automatically to the `gh-pages` branch you can run
 
 ```bash
-$ aegir docs --publish
+$ aegir docs --help
```

Comment:
Is this change intended?  Using the help flag according to the previous statement seems strange

Reply:
I noticed that the docs publish was removed, but perhaps you need to change the sentence before